### DISCUSSION TITLE
Proxmox Updater fails with OPNsense VM

### DISCUSSION THREAD

**Discussion by FugginOld** [tteck/Proxmox#2524] [Help Needed] [answered]
When running proxmox updater it will go down all the line of all the LXCs and as soon as it hit OPNsense VM...it gets an error and stops. Is there something I should look into?

**Comment by tteck** [comment]
OPNsense is a VM, it shouldn't be trying to update it. Does it show in the dialog box to be skipped?

**Reply by FugginOld** [reply]
No...it gives an error and stops before going to the next nodes.

![Screenshot 2024-02-21 154035](https://github.com/tteck/Proxmox/assets/121835359/258d708d-3b2f-423d-a5d0-e35980c98733)

**Reply by tteck** [reply, marked as answer]
### That is **not** a script from this repository!!

**Comment by FugginOld** [comment]
Omg...I am sorry...I use all your scripts and forgot that was BassT23 script...
